prow/e2e-suite.sh

```diff
@@ -69,13 +70,4 @@ mkdir -p ${GOPATH}/src/istio.io/istio/_artifacts
 trap "cp -a /tmp/istio* ${GOPATH}/src/istio.io/istio/_artifacts" EXIT
 
 echo 'Running Integration Tests'
-./tests/e2e.sh ${E2E_ARGS[@]:-} "$@" \
-  --mixer_tag "${GIT_SHA}"\
-  --mixer_hub "${HUB}"\
-  --pilot_tag "${GIT_SHA}"\
-  --pilot_hub "${HUB}"\
-  --ca_tag "${GIT_SHA}"\
-  --ca_hub "${HUB}"\
-  --istioctl ${GOPATH}/bin/istioctl
-
-
+time make e2e_all
```

Comment:
I could easily be misreading things, but various scripts seem to call this script with parameters like "--auth_enable" and "--cluster_wide".  Some of these were getting passed around via the $@.  What used to be a call to "./tests/e2e.sh $@" has now been replaced with a simple "make e2e_all".  It looks to me like these parameters are now being discarded/ignored.  I'm thinking what might be needed is something like this:
make e2e_all TESTOPTS="$@"
or maybe just explicit handling for --auth_enable if --cluster_wide is just intended as a parameter for cluster_lib.sh.